hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/SparkMergeHelper.java

```diff
@@ -77,7 +77,7 @@ public void runMerge(HoodieTable<T, JavaRDD<HoodieRecord<T>>, JavaRDD<HoodieKey>
       readSchema = mergeHandle.getWriterSchemaWithMetaFields();
     }
 
-    BoundedInMemoryExecutor<GenericRecord, GenericRecord, Void> wrapper = null;
+    BoundedInMemoryExecutor<GenericRecord, GenericRecord, Void> executor = null;
```

Comment:
In future, could we do these renames in a separate PR?

Reply:
OK~

Reply:
@vinothchandar i actually asked to do it, while @guanziyue was fixing things here. 

We talked about it, and surely I see your point that this expands the surface for reviewer, but am afraid that small clean ups like that if not done right away, they're just sentenced to not be done ever (since someone will need to consciously set time to come back and clean this up)